pkg/node/volumemgr.go

```diff
@@ -1136,3 +1156,12 @@ func (m *VolumeManager) isPVCNeedFakeAttach(volumeID string) bool {
 
 	return false
 }
+
+func (m *VolumeManager) overrideDriveHealth(drive *api.Drive, overriddenHealth, driveCRName string) {
+	if (overriddenHealth == apiV1.HealthSuspect) || (overriddenHealth == apiV1.HealthBad) {
```

Comment:
What is the purpose of this check? Can't user mark BAD disk as GOOD?

Reply:
Check with HealthGood and HealthUnknown too